Neo4j.Driver/Neo4j.Driver/Experimental/EagerResult.cs

```diff
@@ -24,41 +24,54 @@ namespace Neo4j.Driver.Experimental;
 /// There is no guarantee that anything in Neo4j.Driver.Experimental namespace will be in a next minor version.
 /// Complete result from a cypher query.
 /// </summary>
-public sealed class EagerResult : IReadOnlyList<IRecord>
+public sealed class EagerResult<T>
```

Comment:
Not for this line but the licensing text above. Looks like this is using the older text. There is a newer non dated version that should be used. Can be copied from the top of ExecutableQuery.cs

Reply:
Done.